src/chat/tools/getConnections.ts

```diff
@@ -36,6 +40,42 @@ export class GetConnectionsTool extends BaseLanguageModelTool<IGetConnectionsPar
   foundConnectionTypes: ConnectionType[] = [];
   missingConnectionTypes: ConnectionType[] = [];
 
+  prepareInvocation(
+    options: LanguageModelToolInvocationPrepareOptions<IGetConnectionsParameters>,
+    token: CancellationToken,
+  ): ProviderResult<PreparedToolInvocation> {
+    const { input } = options;
+    let invocationMessage: string;
+    let confirmationMessage: MarkdownString;
+    if (input.connectionType) {
+      invocationMessage = `Get all available ${getConnectionLabel(input.connectionType)} connections`;
+      confirmationMessage = new MarkdownString()
+        .appendMarkdown(`## ${getConnectionLabel(input.connectionType)} Connections\n`)
+        .appendMarkdown(
+          `This tool will look up all available connections of type **${getConnectionLabel(input.connectionType)}**.`,
+        )
+        .appendMarkdown(`Results will show the connection ID and name.`)
+        .appendMarkdown(`Do you want to proceed?`);
```

Comment:
Minor: We should probably combine these into a single `.appendMarkdown` call as well since they're still missing some spacing:
<img width="971" alt="image" src="https://github.com/user-attachments/assets/09ff9e1f-4f62-4f4a-8a8a-e861e47e8874" />